generate/generate.go

```diff
@@ -9,6 +9,11 @@ import (
 	"encoding/pem"
 )
 
+const (
+	PrivateKeyFile = "private.key"
+	PublicKeyFile = "key.pub"
```

Comment:
this is slightly confusing, maybe:
```
autograph.key
autograph.pub
```
and to be sure document somewhere that:
`key` = private key
`pub` = public key a.k.a certificate